### ISSUE TITLE
[HTML5] "method" can not rebind "toString" - but works in Windows

### ISSUE THREAD

**Issue by Grisgram** [YoYoGames/GameMaker-Bugs#5782]
### Description
Hello there,

In HTML5, the "method" function can not rebind "toString" to another struct class.
In Windows, this works.

See attached project.

Start it with Windows target, and you receive this output in the log:
```
------------------ Copying --------------------
Copying 'member'. is_method: 0
Copying 'member_method'. is_method: 1
Copying 'toString'. is_method: 1
------------------ Calling rebound method --------------------
Hello World
From .toString(): Hello World
------------------ Exiting --------------------
```

Start it in HTML5 and you get these crashes:
```
------------------ Copying --------------------
Copying 'toString'. is_method: 1
Copying 'member'. is_method: 0
Copying 'member_method'. is_method: 1
------------------ Calling rebound method --------------------
Hello World
Unhandled Exception - Uncaught { message : "unable to call function undefined typeof=undefined", longMessage : "unable to call function undefined typeof=undefined", stacktrace : [ "function _4f("unable to call function undefined typeof=undefined")
","function _U3([undefined])
","function _q3(global, global)
","function _04()
","function _Sh2([unknown])
","function _yf3(7503.5)
" ], script : "", line : -1 } in file http://127.0.0.1:51264/html5game/html-method-bug.js?cachebust=314580972 at line 199
```

Hope you can find this!
Kind regards, 
Gris

### Steps To Reproduce
* Start the attached project in windows -> all is good
* Start it in HTML -> crash

### Which version of GameMaker are you reporting this issue for?
IDE v2024.4.0.137 Runtime v2024.4.0.168

### Which operating system(s) are you seeing the problem on?
Windows 10.0.19045.0

### Which platform(s) are you seeing the problem on?
HTML5, Windows

[815f383e-1df5-4a05-80fd-3dbcffcf1da0](https://api.gamemaker.io/api/github/downloads/815f383e-1df5-4a05-80fd-3dbcffcf1da0)

**Comment by rwkay** [comment]
Fixed in 2024.6 - `variable_struct_get/set` and ilk did not handle variables that were called the same as JS prototype functions... This commit ensures that the prototype functions are ignored.

**Comment by mgeddesGM** [comment]
a similar error but with a different line number can be seen in  IDE v2024.600.0.569  Runtime v2024.600.0.596
![image](https://github.com/YoYoGames/GameMaker-Bugs/assets/103565197/538c151d-3a03-4df5-a36f-4adbcf2c557b)

**Comment by rwkay** [comment]
Unable to replicate building from current HEAD of 2024.06-main on  GameMaker-HTML5 at 16:05 Friday 07/06/2024

**Comment by Emc1923** [comment]
Verified fixed as of  IDE v2024.6.0.154  Runtime v2024.6.0.204.
